releasenotes/_deconst.json

```diff
@@ -0,0 +1,8 @@
+{
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13”,
+    "githubUrl": "https://github.com/rackerlabs/docs-rackspace/",
```

Comment:
The githubUrl specifies the link provided for github issues when preferGithubIssues is set to 'true' below. Do we want Rel Notes issues filed in rpc-openstack or docs-rackspace?

See https://github.com/rackerlabs/docs-rackspace/blob/master/tools/add-github-links-to-docs.rst